### PR TITLE
Create Italian translation 

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: i18n/fotema.ftl [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+# Mute or unmute audio of a video.
+# Attributes:
+#  .tooltip - Tooltip on mouse hover.
+viewer-mute =
+  .tooltip = Muto/Smuta
+
+# Convert all incompatible videos description
+viewer-convert-all-description = Questo video deve essere convertito prima di poter essere riprodotto. Questo deve accadere solo una volta, ma ci vuole un po' di tempo per convertire un video.
+
+viewer-convert-all-button = Convert all incompatible videos
```
Comment (blissd) [inline i18n/fotema.ftl:113]: Do you want to convert this key, too?

File: i18n/fotema.ftl [hunk 2]
```diff
@@ -0,0 +1,225 @@
+
```
Comment (blissd) [inline i18n/fotema.ftl:1]: Separate `fotema.ftl.license` files are no longer needed for localisation files, so please would you add the SPDX licence tags at the top of the file? For this file format it should look like this:
```
# SPDX-FileCopyrightText: © 2024 David Bliss
#
# SPDX-License-Identifier: GPL-3.0-or-later
```

**Comment by albanobattistella** [comment]
HI,

I have not privileges to do a folder.


Il giorno lun 27 mag 2024 alle ore 13:04 David Bliss <
***@***.***> ha scritto:

**Comment by blissd** [comment]
> HI, I have not privileges to do a folder.

I'm don't know what that means! But I'll merge the PR and move your translation to the right place. Thank you for your PR.

**Comment by blissd** [comment]
Your translation should be in the 1.7.0 release of Fotema, which was published this morning.

**Pull request merged**